lib/mayaUsd/ufe/UsdUINodeGraphNode.cpp

```diff
@@ -48,6 +48,38 @@ void UsdUINodeGraphNode::SetPosOrSizeCommand::executeImplementation()
     }
 }
 
+#ifdef UFE_UINODEGRAPHNODE_HAS_DISPLAYCOLOR
```

Comment:
Pretty much all the new code here came from Lookdex. I only modified the methods to take/return a Ufe::Color3f instead of std::string